packages/web-react/src/components/Tooltip/useTooltipStyleProps.ts

```diff
@@ -13,31 +12,23 @@ export interface UseTooltipStylePropsReturn {
     arrowClassName: string;
     closeButtonClassName: string;
   };
-  props: TooltipProps;
+  props: Omit<TooltipProps, 'id' | 'onToggle'>;
 }
 
-export const useTooltipStyleProps = (props: UseTooltipStyleProps): UseTooltipStylePropsReturn => {
-  const { placement = 'bottom', isDismissible, open, ...modifiedProps } = props;
-
-  useDeprecationMessage({
-    method: 'custom',
-    trigger: placement === 'off',
-    componentName: 'Tooltip',
-    customText:
-      'The "off" value of property "placement" is deprecated and will be removed in the next major version. Use TooltipModern component instead.',
-  });
-
+export const useTooltipStyleProps = (
+  props: Omit<UseTooltipStyleProps, 'id' | 'onToggle'>,
```

Comment:
maybe it would be a good idea to store `'id' | 'onToggle'` in some variable/const? Because when you add a new omitted prop, you would have to add it twice now.

Reply:
good idea, created new type